tutorials/circuits/3_summary_of_quantum_operations.ipynb

```diff
@@ -228,6 +228,22 @@
     "job.result().get_unitary(qc, decimals=3)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Note on U gate deprecation\n",
+    "\n",
+    "The QuantumCircuit methods $u1$, $u2$ and $u3$ are now deprecated. Instead, the following replacements should be used.\n",
+    "\n",
+    "$u1(\\lambda) = p(\\lambda) = u(0, 0, \\lambda)$\n",
+    "\n",
+    "$u2(\\phi, \\lambda) = u(\\frac{\\pi}{2}, \\phi, \\lambda) = p(\\frac{\\pi}{2} + \\phi) \\cdot sx \\cdot p(\\frac{\\pi}{2} - \\lambda)$\n",
+    "\n",
+    "$u3(\\theta, \\phi, \\lambda) = u(\\theta, \\phi, \\lambda) = p(\\phi + \\pi) \\cdot sx \\cdot p(\\theta + \\pi) \\cdot sx \\cdot p(\\lambda)$\n",
+    "\n"
+   ]
+  },
```

Comment:
Can you add code examples (no need to put in a code cell, just put as code in markdown) so that it's easier for other people to migrate their old code? Something like

```python
# qc.u1(lambda) is now:
qc.p(lambda)

# qc.u2(phi, lambda) is now:
qc.u(pi/2, phi, lambda)

# qc.u3(theta, phi, lambda) is now:
qc.u(theta, phi, lambda)

```

Reply:
Added in. Should be completed.